src/operations/run_command.ts

```diff
@@ -51,6 +51,7 @@ export class RunAdminCommandOperation<T = Document> extends AbstractOperation<T>
   constructor(
     public command: Document,
     public override options: RunCommandOptions & {
+      /** @deprecated Will be removed in the next major version, and replaced with an option to set writeConcern.w to 0 */
       noResponse?: boolean;
```

Comment:
This isn't publicly viewable, so if it was possible to transition this to using w=0 we could remove this from here. Close this if we don't go with my question about checking options.